ivy-hydra.el

```diff
@@ -34,11 +34,11 @@
 (require 'hydra)
 
 (defun ivy--matcher-desc ()
-  "Return description of `ivy--regex-function'."
-  (if (eq ivy--regex-function
-          'ivy--regex-fuzzy)
-      "fuzzy"
-    "ivy"))
+  "Return description of `ivy--regex-function'. "
+  (let ((cell (assoc ivy--regex-function ivy--preferred-re-builders)))
+    (if cell
+        (cdr cell)
```

Comment:
I would keep the formatting out of this function, since conceivably users might use this display name in other ways. The hydra-ivy hint is formatted with a min width of 5 padded on the right. A longer string here does mean the last column doesn't column won't line up but it still seems readable.